src/main/java/nextstep/subway/domain/Line.java

```diff
@@ -0,0 +1,57 @@
+package nextstep.subway.domain;
+
+import lombok.AccessLevel;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import nextstep.subway.presentation.LineRequest;
+
+import javax.persistence.*;
+
+@NoArgsConstructor(access = AccessLevel.PROTECTED)
+@Getter
+@Entity
+public class Line {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+
+    @Column(length = 20, nullable = false)
+    private String name;
+
+    @Column(length = 20, nullable = false)
+    private String color;
+
+    @Embedded
+    private Sections sections = new Sections();
+
+    public Line(String name, String color) {
+        this.name = name;
+        this.color = color;
+    }
+
+    public static Line createLine(Station upStation, Station downStation, LineRequest lineRequest) {
+        Line createdLine = new Line(lineRequest.getName(), lineRequest.getColor());
+
+        createdLine.addSection(Section.createSection(
+                createdLine,
+                upStation,
+                downStation,
+                lineRequest.getDistance()
+        ));
+
+        return createdLine;
+    }
+
+    public void addSection(Section section) {
+        Sections sections = this.getSections();
+        sections.addSection(section);
```

Comment:
getter로 가져올 필요 없이 직접 참조해도 될 것 같습니다.
```suggestion
        sections.addSection(section);
```